File: Hoteles-Reserva-WS/src/java/models/PackageInfo.java
package models;

public class PackageInfo {
    private int idPackages;
    private String name;
    private Boolean active;

    public int getIdPackages() {
        return idPackages;
    }

    public void setIdPackages(int idPackages) {
        this.idPackages = idPackages;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Boolean getActive() {
        return active;
    }

    public void setActive(Boolean active) {
        this.active = active;
    }
    
    
}

File: Hotel-Reservacion/Hotel-Reservacion/Models/PackageInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Hotel_Reservacion.Models
{
    public class PackageInfo
    {
        private String name;
        private bool active;
        private int idPackages;

        public String Name
        {
            get
            {
                return name;
            }
            set
            {
                name = value;
            }
        }
        public int IdPackages
        {
            get
            {
                return idPackages;
            }
            set
            {
                idPackages = value;
            }
        }
        public bool Active
        {
            get
            {
                return active;
            }
            set
            {
                active = value;
            }
        }
    }
}